src/node/desktop/src/main/gwt-callback.ts

```diff
@@ -362,7 +362,10 @@ export class GwtCallback extends EventEmitter {
     });
 
     ipcMain.on('desktop_show_ppt_presentation', (event, pptDoc: string) => {
-      GwtCallback.unimpl('desktop_show_ppt_presentation');
+      shell.openPath(resolveAliasedPath(normalizeSeparatorsNative(pptDoc))).catch((value) => {
```

Comment:
Same here where we can show it in the View pane if there's an error.

Reply:
Looks like it's fine here after the discussion with opening docx files. There's no need for a fallback.